packages/carbon-web-components/.storybook/main.js

```diff
@@ -29,9 +27,19 @@ module.exports = {
   addons: [
     '@storybook/addon-actions',
     '@storybook/addon-docs',
+    {
+      name: '@storybook/addon-essentials',
+      options: {
+        actions: true,
+        backgrounds: false,
+        controls: true,
```

Comment:
probably should turn off controls for now
```suggestion
        controls: false,
```

Reply:
I actually purposely left them on in case we want to experiment with transitioning over. They don't seem to interfere with anything while be active along side knobs?